Neos.Flow/Classes/Http/RequestHandler.php

```diff
@@ -167,6 +167,8 @@ protected function sendResponse()
             header($prepareHeader);
         }
         echo $response->getBody()->getContents();
-        ob_end_flush();
+        while (ob_get_level() > 0) {
```

Comment:
I guess this also makes it technically impossible to catch a `sendResponse()` call into an output buffer now. Not that this makes much sense inside the HttpRequestHandler anyway, so fine for me

Reply:
I can't even remember when I last used anything dealing output buffering…

Reply:
me neither 🤷‍♂